### ISSUE TITLE
Accessibility: Ctrl+Down/UpArrows does not work in Tree find control

### ISSUE THREAD

**Issue by vscodenpa** [vscodenpa/testissues#21655]
CC -meganrogge -isidorn

1. Open File Explorer (Ctrl+Shift+E).

1. Ctrl+F to open Tree find control and type search term.

1. Press Ctrl+DownArrow to see if you can jump to the search result: not working.

1. In the search result, press Ctrl+UpArrow to see if you can go back to search input: not working.
